### ISSUE TITLE
Rule Style and Color not coming through

### ISSUE THREAD

**Issue by captDaylight** [markusantonwolf/tailwind-css-plugin-multi-columns#5]
My class is:
```
class="column-3 column-gap-5 rule-solid rule-gray-900"
```

The columns are working as expected but the rule is not appearing. If I look in console I see:
<img width="388" alt="Screen Shot 2021-06-03 at 3 10 19 PM" src="https://user-images.githubusercontent.com/700617/120705977-7e62d480-c486-11eb-9545-faa4b4f4a49f.png">

If I remove the appended `--ta-` from `.rule-solid` the rule will appears.

Great work on this plugin!

**Comment by markusantonwolf** [comment]
@captDaylight the easiest solution would be to define the with of the rule like this:

`class="column-3 column-gap-5 rule-2 rule-gray-900"` for a rule-width of 2px or
`class="column-3 column-gap-5 rule rule-gray-900"` for a rule-width of 1px.

rule-X

The Plugin uses the Tailwind CSS [border width definitions](https://tailwindcss.com/docs/border-width) in the theme to render the rule width like:

## rule 
```
    --ta-column-rule-style: solid;
    --ta-column-rule-color: black;
    --ta-column-rule-opacity: 1;
    --ta-column-rule-width: 1px;
    -moz-column-rule: var(--ta-column-rule-width) var(--ta-column-rule-style) var(--ta-column-rule-color);
    column-rule: var(--ta-column-rule-width) var(--ta-column-rule-style) var(--ta-column-rule-color);
```

## rule-2
```
    --ta-column-rule-style: solid;
    --ta-column-rule-color: black;
    --ta-column-rule-opacity: 1;
    --ta-column-rule-width: 2px;
    -moz-column-rule: var(--ta-column-rule-width) var(--ta-column-rule-style) var(--ta-column-rule-color);
    column-rule: var(--ta-column-rule-width) var(--ta-column-rule-style) var(--ta-column-rule-color);
```

**Comment by markusantonwolf** [comment]
@captDaylight but checking your feedback I was able to find an other bug I fixed immediately - so thank you :-)

**Comment by captDaylight** [comment]
Thanks for the clarification!

**Comment by captDaylight** [comment]
One more question, when I use `rule rule-dashed rule-opacity-50` I'm seeing that `rule-dashed` is overriding `rule`, but not `rule-opacity-50`. I'm seeing the same thing when I was trying to change the color.

<img width="457" alt="Screen Shot 2021-06-04 at 2 11 57 PM" src="https://user-images.githubusercontent.com/700617/120851710-35bf2000-c547-11eb-9f70-30af3173ff47.png">

**Comment by markusantonwolf** [comment]
@captDaylight - well that's exactly the bug I fixed in 8e86f519e85c99ff242a146cefae791b1f00a0f9 - the order was wrong. Just update to version 1.1.3 😃 

<img width="400" src="https://user-images.githubusercontent.com/1286725/120879764-3cf23680-c5f8-11eb-8ad5-322b59524d56.png">

**Comment by captDaylight** [comment]
ah, gotcha. Thanks!
